### ISSUE TITLE
using ESM - model methods are broken

### ISSUE THREAD

**Issue by dcsan** [sequelize/sequelize-typescript#1369]
# Issue 
when trying to use ES Modules, all of the default Model methods are missing from my classes.

![image](https://user-images.githubusercontent.com/514002/180635845-d49a52ab-31b1-43d1-9724-84b074f279af.png)

interestingly the `asset.save` method doesn't throw an error?

where Asset is just a model:

```ts
@Table
export class Asset extends Model {
```

## Versions

- sequelize:
- sequelize-typescript:
- typescript:

```
├─┬ sequelize-typescript@2.1.3
│ └── sequelize@6.17.0 deduped
└── sequelize@6.17.0

└── sequelize-typescript@2.1.3

├─┬ ts-node@10.9.1
│ └── typescript@4.7.4 deduped
├─┬ ts-patch@2.0.1
│ └── typescript@4.7.4 deduped
└── typescript@4.7.4

```

## Issue type

- [X] bug report
- [ ] feature request

## Actual behavior

cannot call basic sequelize methods on any model such as `MyModel.truncate()` or `MyModel.create()`

where `MyModel` is a subclass of `Model`

## Expected behavior

should be able to create!

## Steps to reproduce

have to use ESM modules

## Related code

This is all configuration bullshit. I'm putting my tsconfig and package.json below.

```json

{
    "compilerOptions": {
        "rootDir": "src",
        "outDir": "./build",
        "lib": [
            "ESNext"
        ],
        "module": "NodeNext",
        "moduleResolution": "NodeNext",
        "target": "es2022",
        "esModuleInterop": true,
        "types": [
            "node"
        ],
        "sourceMap": true,
        "strict": true,
        "strictPropertyInitialization": false,
        "baseUrl": "./src",
        "paths": {
            "@sharedTypes/*": [
                "../../client/src/types/shared/*"
            ]
        },
        "experimentalDecorators": true,
        "emitDecoratorMetadata": true,
        "skipLibCheck": true,
        "forceConsistentCasingInFileNames": true,
        "resolveJsonModule": true,
        "allowJs": false
    },
    "include": [
        "src/**/*",
        "../../client/src/types/**/*"
    ],
    "exclude": [
        "node_modules"
    ],
    "typedocOptions": {
        "entryPoints": [
            "src/server.ts"
        ],
        "out": "docs/build"
    },
    "ts-node": {
        "esm": true,
        "experimentalSpecifierResolution": "node",
        "require": [
            "tsconfig-paths/register"
        ]
    }
}

```
and top of package.json
```json

{
    "name": "plugbot-server",
    "version": "0.0.3",
    "description": "A framework for extensible crypto related discord bots with plugins",
    "main": "index.js",
    "type": "module",

...

    "nodemonConfig": {
        "ignore": [
            "*.json"
        ],
        "events": {
            "start": "clear || cls"
        }
    }


```
